src/components/registration/steps.jsx

```diff
@@ -752,7 +759,7 @@ module.exports = {
                                            id: 'registration.validationMaxLength'
                                        })
                                    }}
-                                   required={(this.state.countryChoice === 'us')} />
+                                   required={(this.state.countryChoice === 'us') ? true : 'isFalse'} />
```

Comment:
Er, I guess the real question is why is that broken, requiring you to use `isFalse`?